docs/guide/configuration.md

```diff
@@ -53,7 +55,52 @@ And now you'll be able to write queries like this:
 const query = myDb.rawQuery(sql`SELECT * FROM users`);
 ```
 
-## 2. A basic example of migrations folder
+## Example 2: Multiple databases connected to your apps
+
+Connect using multiple databases and credentials used by multiple apps
+
+::: tip DEMO
+Check out [@ts-safeql-demos/basic](https://github.com/ts-safeql/safeql/tree/main/demos/basic) for a working example.
```

Comment:
Incorrect demo link. Should point to demos/multi-connections

Reply:
eb993fe559a200fa8890c83c32f3789a802d2faa

Reply:
Sorry about that! Totally overlooked these demos!